themes/janeswalk/js/components/pages/Dashboard/DashboardSummary.jsx

```diff
@@ -0,0 +1,29 @@
+
+//TODO: Need to find or solidify the data source for impact data - some items not covered (in dashboard or impact static data) for below recap (Post PR)
+//TODO: Bring in <SingleBarGraph/> (Post PR)
+//TODO: Place all data collections items within the Store, remove reduces from below - also summary is not working, so remove this from here (Post PR)
+//TODO: total walk leaders: ${impact.walkLeaders.reduce((p,c)=>(p.total + c.total),0)} (Post PR)
+//TODO: total walks: ${impact.walks.reduce((p,c)=>(p.total + c.total),0)} (Post PR)
+//TODO: Place all .length calls in Store instead. (Post PR)
+
+const DashboardSummary = ({name, walkLeaders, walks, users, impact}) => {
+  debugger;
+  return (
+    <section>
+      <h2>Recap</h2>
+      <h4>{`${walkLeaders.length} ${name} walk leaders led ${walks.length} walks as a part of Jane's Walk ${(new Date()).getFullYear()} reaching more than ${users.length} participants` }</h4>
```

Comment:
Plurals can use the `t2()` i18n translations function, which assigns the correct plural. If a city has had only 1 walk leader, it needs to be singular.